helm/forgerock-metrics/README.md

```diff
@@ -57,10 +68,10 @@ A PrometheusRules CRD has been included in the Helm chart which includes the fr-
 * Deployed ForgeRock application in Google Cloud cluster.
 * Kubectl authenticated to cluster.
 * Separate namespace prepared for deploying Prometheus/Grafana
-(Use namespace that is different to where your application is running).
+(Use namespace that is different to where your application is running, default is 'monitoring').
```

Comment:
s/Use namespace/Use a namespace/
s/running, default is 'monitoring'/running. The default is 'monitoring'./